src/neo/SmartContract/Native/Tokens/NeoToken.cs

```diff
@@ -87,19 +91,8 @@ private BigInteger CalculateBonus(BigInteger value, uint start, uint end)
 
         internal override void Initialize(ApplicationEngine engine)
         {
-            BigInteger amount = TotalAmount;
-            for (int i = 0; i < Blockchain.StandbyCommittee.Length; i++)
-            {
-                ECPoint pubkey = Blockchain.StandbyCommittee[i];
-                RegisterCandidateInternal(engine.Snapshot, pubkey);
-                BigInteger balance = TotalAmount / 2 / (Blockchain.StandbyValidators.Length * 2 + (Blockchain.StandbyCommittee.Length - Blockchain.StandbyValidators.Length));
-                if (i < Blockchain.StandbyValidators.Length) balance *= 2;
-                UInt160 account = Contract.CreateSignatureRedeemScript(pubkey).ToScriptHash();
-                Mint(engine, account, balance);
-                VoteInternal(engine.Snapshot, account, pubkey);
-                amount -= balance;
-            }
-            Mint(engine, Blockchain.GetConsensusAddress(Blockchain.StandbyValidators), amount);
+            engine.Snapshot.Storages.Add(CreateStorageKey(Prefix_VotersCount), new StorageItem(new byte[0]));
```

Comment:
```suggestion
            engine.Snapshot.Storages.Add(CreateStorageKey(Prefix_VotersCount), new StorageItem(Array.Empty<byte>()));
```

Reply:
Both `System` and `Neo.VM.Types` are imported.